app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class UserPresenter < BasePresenter
```

Comment:
No need to add a UserPresenter here as it is not used anywhere yet. I am also trying to get rid of presenters and just use ViewComponent

Reply:
Removed commit.